applications/conga-cards/.eslintrc.js

```diff
@@ -0,0 +1,99 @@
+module.exports = {
```

Comment:
check some of the other node.js typescript projects, eg runhfsc etc.   there's a way of short cutting a lot of this file.

stylistic choice, but it can be useful.